package.json

```diff
@@ -35,16 +35,16 @@
     "less": "^4.2.0",
     "less-loader": "^12.2.0",
     "mini-css-extract-plugin": "^2.9.1",
-    "node-polyfill-webpack-plugin": "^3.0.0",
+    "node-polyfill-webpack-plugin": "^4.0.0",
     "postcss": "^8.4.45",
     "postcss-flexbugs-fixes": "^5.0.2",
     "postcss-global-import": "^1.0.6",
     "postcss-loader": "^8.1.1",
-    "postcss-normalize": "^10.0.1",
-    "postcss-preset-env": "^9.6.0",
+    "postcss-normalize": "^13.0.1",
+    "postcss-preset-env": "^10.1.0",
```

Comment:
I am a bit concerned about the nesting change mentioned here: https://github.com/csstools/postcss-plugins/wiki/PostCSS-Preset-Env-10  . This might cause styling differences in our components
Unfortunately, I don't think we can verifiy this manually in detail by only using storybook.
So, we need to check it using screenshot tests. This requires upgrading  postcss-preset-env in enact/cli and use that cli branch to check for differences.